examples/tutorials/05_ordinary_models.ipynb

```diff
@@ -0,0 +1,382 @@
+{
```

Comment:
<p>Either use the same style of plots, as in other tutorials, or update rest of them to use this one</p><p>Also, you can use <code>after</code> pipeline to plot loss</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>